libs/expo/betterangels/src/lib/screens/Client/Profile/PersonalInfo.tsx

```diff
@@ -36,6 +38,12 @@ export default function PersonalInfo(props: IProfileSectionProps) {
     client?.clientProfile.user.lastName ?? ''
   }`.trim();
 
+  const displayDob = client?.clientProfile.dateOfBirth
+    ? `${format(client?.clientProfile.dateOfBirth, 'MM/dd/yyyy')} (${
```

Comment:
Optional. But unless it's a simple one-liner i'd prefer to avoid ternarys, so something like this?

```javascript
   const formattedDob = format(client?.clientProfile.dateOfBirth, 'MM/dd/yyyy') || null;
   const clientAge = client?.clientProfile.age;

   const displayDob = formattedDob && clientAge && `${formattedDob} ${clientAge}`
```